### ISSUE TITLE
[Question] some basic question

### ISSUE THREAD

**Issue by cxchen100** [tensorflow/adanet#105]
I have use adanet in my tensorflow code which have two candidate model_A and model_B.

by the tensorboard  auc curve , find model_A  is better then model_B

now I am not sure the exporter is the best model 's ouput or combine two model's effective ? 

If I output this tf.estimator 's export to the production .  what may get ?

**Comment by cweill** [comment]
@cxchen100 In your TensorBoard you'll see three kinds of curves: `.*subnetwork.*`, `.*ensemble.*`, and a couple like `<model_dir>/eval`. The `.*ensemble.*` models are the only candidates that will be considered for serving. AdaNet tracks the moving average of their training loss to determine which is the best candidate at any given step. The  `<model_dir>/eval` line in TensorBoard follows the current 'best' candidate according to AdaNet.
